### DISCUSSION TITLE
What are the info endpoints labeled?

### DISCUSSION THREAD

**Discussion by ag-gaphp** [Stirling-Tools/Stirling-PDF#1617] [Q&A] [answered]
This is the list I have for my `ENDPOINTS_TO_REMOVE` variable in Docker.

```
adjust-contrast
crop
extract-page
multi-page-layout
pdf-organizer
pdf-to-single-page
rotate-pdf
scale-pages
file-to-pdf
img-to-pdf
pdf-to-html
pdf-to-img
pdf-to-pdfa
pdf-to-markdown
pdf-to-presentation
pdf-to-text
pdf-to-xml
xlsx-to-pdf
cert-sign
remove-cert-sign
change-permissions
remove-password
sanitize-pdf
add-image
add-page-numbers
auto-rename
change-metadata
compare
compress-pdf
extract-image-scans
extract-images
flatten
get-info-on-pdf
ocr-pdf
remove-blanks
repair
show-javascript
sign
```

However, I cannot hit the `/api/v1/info/status` endpoint. I get the following message:

```
This endpoint is disabled.
```

I don't see anything specific for the status on this page: https://github.com/Stirling-Tools/Stirling-PDF/blob/main/Endpoint-groups.md

I would think you shouldn't be able to disable this particular endpoint, but somehow I have. Just curious which of these options I need to remove from my list.

edit: It's not just the status endpoint, it appears to be all of my Info API endpoints. I also get a disabled message when I try to hit any of the others listed in the Swagger UI API reference for my instance.

![image](https://github.com/user-attachments/assets/baa9a71b-9e3b-450a-84cf-944dfb0587e7)

**Comment by Frooodle** [comment, marked as answer]
There is a setting you have enabled either in docker or config.yml
To disable metrics 
Please check 

Endpoint disabling is for UI side

**Reply by ag-gaphp** [reply]
Yup. I was focused on the endpoint wording. Didn't RTFM and conflated metrics = telemetry when I made the compose file. Thanks.
